src/Api/Vault/Controllers/SyncController.cs

```diff
@@ -7,10 +7,11 @@
 using Bit.Core.Repositories;
 using Bit.Core.Services;
 using Bit.Core.Settings;
+using Bit.Core.Vault.Repositories;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
-namespace Bit.Api.Controllers;
+namespace Bit.Api.Vault.Controllers;
```

Comment:
Do the sync response belong to Vault? It also includes data outside of vault.

Reply:
Yes, according to [this sheet](https://docs.google.com/spreadsheets/d/1tPiAWux36-d1-Po10R3wICfOgr9JWhZLa76I-MInauA/edit#gid=1473123964) sync belongs to Vault. I'll make a ticket to look into how we can draw these lines for data outside of Vault's domain.